shelley/chain-and-ledger/shelley-spec-ledger-test/shelley-spec-ledger-test.cabal

```diff
@@ -115,6 +117,10 @@ test-suite shelley-spec-ledger-test
       Test.Shelley.Spec.Ledger.Examples.Updates
       Test.Shelley.Spec.Ledger.Fees
       Test.Shelley.Spec.Ledger.MultiSigExamples
+      Test.Shelley.Spec.Ledger.NonTraceProperties.Generator
```

Comment:
While you were making these changes originally, I went to some trouble to remove the NonTraceProperties (and Hedgehog too). Rather than using these generators you can use one of the trace generators (which generate richer data) and remove NonTraceProperties.*

Reply:
We can try this in another PR.  Don't want to mix things up too much.